app/mainwindow2.cpp

```diff
@@ -970,7 +970,7 @@ void MainWindow2::makeConnections(Editor* editor, ScribbleArea* scribbleArea)
 {
     connect(editor->tools(), &ToolManager::toolChanged, scribbleArea, &ScribbleArea::setCurrentTool);
     connect(editor->tools(), &ToolManager::toolPropertyChanged, scribbleArea, &ScribbleArea::updateToolCursor);
-    connect(editor->layers(), &LayerManager::currentLayerChanged, scribbleArea, &ScribbleArea::updateAllFrames);
+    connect(editor->layers(), QOverload<int>::of(&LayerManager::currentLayerChanged), scribbleArea, &ScribbleArea::updateAllFrames);
```

Comment:
Sorry I don't quite understand. Could you tell what `QOverload` is?

Reply:
`BaseManager` has the function `currentLayerChanged(Layer*)`, and its child class `LayerManager` has `currentLayerChanged(int)`, so the `QOverload<int>::of()` is used to specify that you want the `currentLayerChanged` function with an `int` parameter.

It's a relatively new feature, so [this](https://doc.qt.io/qt-5/qtglobal.html#qOverload) is about the best documentation Qt has on it at the moment.